We should investigate if it is really needed. It came with first
integration into Solaris (PSARC nor bug have no info why these were
included).

--- src/Makefile.am
+++ src/Makefile.am
@@ -34,12 +34,12 @@
 bin_SCRIPTS = bdftogd
 
 if HAVE_LIBZ
-check_PROGRAMS += gd2time
+bin_PROGRAMS += gd2time
 endif
 
 EXTRA_DIST = bdftogd demoin.png entities.html entities.tcl CMakeLists.txt config.h.cmake gd_io_stream.cxx gdpp.cxx msinttypes/inttypes.h msinttypes/stdint.h
 
-include_HEADERS = gd.h gdfx.h gd_io.h gdcache.h gdfontg.h gdfontl.h gdfontmb.h gdfonts.h gdfontt.h entities.h gd_color_map.h gd_errors.h gdpp.h
+include_HEADERS = gd.h gdfx.h gd_io.h gdcache.h gdfontg.h gdfontl.h gdfontmb.h gdfonts.h gdfontt.h entities.h gd_color_map.h gd_errors.h gdpp.h gdhelpers.h jisx0208.h wbmp.h
 
 lib_LTLIBRARIES = libgd.la
 
